deployment/terraform/metrics.go

```diff
@@ -312,6 +312,24 @@ func (t *Terraform) setupMetrics(extAgent *ssh.ExtAgent) error {
 
 	mlog.Info("Setting up Grafana", mlog.String("host", t.output.MetricsServer.GetConnectionIP()))
 
+	// Change Grafana admin password
+	// No need to be cryptographically secure here, we just need a new password
+	passLen := 32
+	chars := []rune("abcdefghijklmnopqrstuvwxyz" + "ABCDEFGHIJKLMNOPQRSTUVWXYZ" + "0123456789")
+	s := make([]rune, passLen)
+	for j := 0; j < passLen; j++ {
+		s[j] = chars[rand.Intn(len(chars))]
+	}
+	password := string(s)
```

Comment:
This could be refactored to a util

Reply:
Done in https://github.com/mattermost/mattermost-load-test-ng/pull/911/commits/2456d87b04074229e42b5be7c4d3f548852c0640